src/uct/api/uct_def.h

```diff
@@ -360,7 +360,16 @@ typedef struct uct_cm_ep_client_connect_args {
     const uct_cm_remote_data_t *remote_data;
 
     /**
-     * Indicates the server's @ref ucs_status_t status.
+     * Indicates the server's @ref ucs_status_t status:
+     * UCS_OK                   - the server accepted the client's connection
+     *                            establishment request.
+     * UCS_ERR_REJECTED         - the server rejected the client's connection
+     *                            establishment request.
+     * UCS_ERR_CONNECTION_RESET - the server's connection was reset during
+     *                            the connection establishment to the client.
```

Comment:
could this really happen? shouldn't it be reject?

Reply:
if the server already accepted the connection request but then died beore the client called its connect cb

Reply:
if it accepted, i would expect this callback will be called with UCS_OK, and later disconnect callback will be called